packages/app-project/src/screens/ProjectHomePage/components/ZooniverseTalk/components/RecentSubjects/helpers/fetchRecentSubjects/getSubjectIdsFromTalkData.js

```diff
@@ -0,0 +1,14 @@
+import flow from 'lodash/fp/flow'
+import map from 'lodash/fp/map'
+import slice from 'lodash/fp/slice'
+import uniq from 'lodash/uniq'
+
+function getSubjectIdsFromTalkData (comments) {
+  return flow(
+    map(comment => comment.focus_id),
+    uniq,
+    slice(0, 3)
```

Comment:
I'm fine with lodash being used for helper functions that don't have a native equivalent, but if we could use the native Array `map` and `slice`, I'd prefer it.

Reply:
These aren't native Array methods, they're the FP versions. It allows them to be composed together to perform multiple transforms without method chaining / lots of clunky temporary variables.